### PR TITLE
OCM-4064: Added new `rhcs_info` data source

### PR DESCRIPTION
**Pull request by sagidayan**
This PR also addresses breaking changes in TF 1.6.0. Pinned CI to 1.5.7

TODO: investigate breakage and support 1.6.0 as well

### CODE REVIEW

**Comment by sagidayan** [comment]
/hold

Until we are fine with doc language.

File: provider/info/info_data_source.go [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+			"organization_name": schema.StringAttribute{
+				Description: "OCM account organization name",
+				Computed:    true,
+			},
+			"ocm_api": schema.StringAttribute{
+				Description: "OCM API url",
+				Computed:    true,
+			},
+			"ocm_aws_account_id": schema.StringAttribute{
+				Description: "OCM AWS account ID",
```
Comment (bardielle) [inline provider/info/info_data_source.go:84]: It's a little bit confusing- what is the difference between the "account_id" attribute -
In my opinion 
account_id -> ocm_account_id
ocm_aws_account_id   -> user_account_id

File: provider/info/info_data_source.go [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+func (d *OCMInfoDataSource) Metadata(ctx context.Context, req datasource.MetadataRequest, resp *datasource.MetadataResponse) {
+	resp.TypeName = req.ProviderTypeName + "_info"
+}
+
+func (d *OCMInfoDataSource) Schema(ctx context.Context, req datasource.SchemaRequest, resp *datasource.SchemaResponse) {
+	resp.Schema = schema.Schema{
+		Description: "",
+		Attributes: map[string]schema.Attribute{
+			"account_id": schema.StringAttribute{
+				Description: "OCM account ID",
```
Comment (bardielle) [inline provider/info/info_data_source.go:52]: The description is not clear enough - maybe 
User account id ?

File: provider/info/info_data_source.go [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+	resp.Diagnostics.Append(diags...)
+}
+
+func extractOCMAWSAccount(url string) string {
+	env := ocmEnvProd
+	if strings.Contains(url, "stage") {
+		env = ocmEnvStage
+	} else if strings.Contains(url, "integration") {
+		env = ocmEnvInt
+	}
```
Comment (bardielle) [inline provider/info/info_data_source.go:148]: URL (connection.URL()) will always return a value, right?

Comment (sagidayan) [reply provider/info/info_data_source.go:148]: yes, its part of the provider (client) setup

File: provider/info/info_data_source.go (hunk 2 above)
Comment (sagidayan) [reply provider/info/info_data_source.go]: Fixed

File: provider/info/info_data_source.go (hunk 1 above)
Comment (bardielle) [reply provider/info/info_data_source.go:84]: OK, So just update the description

File: Makefile [hunk 4]
```diff
@@ -117,6 +117,7 @@ e2e_test: tools install
 
 .PHONY: apply_folder
 apply_folder: install
+	sleep 1h
```
Comment (bardielle) [inline Makefile:120]: Remove it please

**Pull request merged**